### ISSUE TITLE
[TF2] MvM specific voices lines are broken  and don't llay anymore.

### ISSUE THREAD

**Issue by condor00fr** [ValveSoftware/Source-1-Games#4138]
Durring a game of MVM, heavies, solders, medics and Engineers had each a unique set of voices lines for different events such as:

-Calling out a sentry buster. (engie only)
-Finishing a wave.
-Encouraging to buy upgrade.
-Encouraging to pick up money.
-Being the last one alive.
-taunts
Calling out snipers (I do believe this one still works?)

Uptate after uptates, they suddenly stopped playing.

Additionnally,  all prementionned classes have an unused set of voices lines for when sapping a robot, presumably meant for a disguised spy. It would be cool to have them reimplemeted along various unused announcer lines.

**Comment by rabscootle** [comment]
Sentry Buster and Sniper callouts work. The remaining never worked, the voice response rules have existed since MvM's launch but the concepts (what triggers them in-game) are never called for in the game code.

**Comment by condor00fr** [comment]
nope, they did worked at some point, problem is they stopped working so long ago for whatever reason that everybody thinks they never did, but thoses who remember them can tell you they used to work.
